timelock-impl/src/main/java/com/palantir/atlasdb/timelock/TimeLockResource.java

```diff
@@ -35,62 +35,62 @@
+    public LockService getLockService(@Safe @PathParam("namespace") String namespace) {
+        return getOrCreateServices(namespace).getLockService();
     }
 
     @Path("/timestamp")
-    public TimestampService getTimeService(@Safe @PathParam("client") String client) {
-        return getOrCreateServices(client).getTimestampService();
+    public TimestampService getTimeService(@Safe @PathParam("namespace") String namespace) {
+        return getOrCreateServices(namespace).getTimestampService();
     }
 
     @Path("/timelock")
-    public Object getTimelockService(@Safe @PathParam("client") String client) {
-        return getOrCreateServices(client).getTimelockService().getPresentService();
+    public Object getTimelockService(@Safe @PathParam("namespace") String namespace) {
+        return getOrCreateServices(namespace).getTimelockService().getPresentService();
     }
 
     @Path("/timestamp-management")
-    public TimestampManagementService getTimestampManagementService(@Safe @PathParam("client") String client) {
-        return getOrCreateServices(client).getTimestampManagementService();
+    public TimestampManagementService getTimestampManagementService(@Safe @PathParam("namespace") String namespace) {
+        return getOrCreateServices(namespace).getTimestampManagementService();
     }
 
     @VisibleForTesting
-    TimeLockServices getOrCreateServices(String client) {
-        return servicesByClient.computeIfAbsent(client, this::createNewClient);
+    TimeLockServices getOrCreateServices(String namespace) {
+        return servicesByNamespace.computeIfAbsent(namespace, this::createNewClient);
     }
 
-    private TimeLockServices createNewClient(String client) {
-        Preconditions.checkArgument(!client.equals(PaxosTimeLockConstants.LEADER_ELECTION_NAMESPACE),
+    private TimeLockServices createNewClient(String namespace) {
+        Preconditions.checkArgument(!namespace.equals(PaxosTimeLockConstants.LEADER_ELECTION_NAMESPACE),
                 "The client name '%s' is reserved for the leader election service, and may not be "
                         + "used.",
                 PaxosTimeLockConstants.LEADER_ELECTION_NAMESPACE);
 
-        if (servicesByClient.size() >= maxNumberOfClients.get()) {
+        if (servicesByNamespace.size() >= maxNumberOfNamespaces.get()) {
             log.error(
                     "Unable to create timelock services for client {}, as it would exceed the maximum number of "
                             + "allowed clients ({}). If this is intentional, the maximum number of clients can be "
                             + "increased via the maximum-number-of-clients runtime config property.",
-                    SafeArg.of("client", client),
-                    SafeArg.of("maxNumberOfClients", maxNumberOfClients));
+                    SafeArg.of("client", namespace),
+                    SafeArg.of("maxNumberOfNamespaces", maxNumberOfNamespaces));
```

Comment:
This might be a little awkward but I suggest leaving this as `client`, to match the user-facing wording

Reply:
Fixed.